internal/config/yaml_parser_test.go

```diff
@@ -81,6 +84,9 @@ func (t *YamlParserTest) TestReadConfigFile_ValidConfig() {
 	AssertEq(ERROR, mountConfig.LogConfig.Severity)
 	AssertEq("/tmp/logfile.json", mountConfig.LogConfig.FilePath)
 	AssertEq("text", mountConfig.LogConfig.Format)
+	AssertEq(100, mountConfig.LogConfig.LogRotate.MaxSizeInMB)
+	AssertEq(5, mountConfig.LogConfig.LogRotate.BackupCount)
+	AssertEq(false, mountConfig.LogConfig.LogRotate.Compress)
```

Comment:
Without documentation, it's not clear why this case should fail. Please either add a comment here to say why, or headers where the config are defined, or at both places.